src/client/components/styling/login.css

```diff
@@ -0,0 +1,74 @@
+.app-login {
+    height: 100%;
+    width: calc(100vw + 7vw);
+    position: absolute;
+    left: -7vw;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background: rgba(207, 216, 220, 0.75);
+}
+
+.login-aside {
+    position: absolute;
+    width: 11rem;
+    margin: 7rem 1rem;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    align-items: center;
+}
+
+.login-container {
+    overflow: hidden;
+    height: 24rem;
+    width: 24rem;
+    background-color: var(--clr-black-1);
+}
+
+.login-aside-text {
+    color: var(--clr-white);
+}
+.login-aside-title {
+    font-size: 3rem;
+}
+
+.login-button {
+    height: 2.5rem;
```

Comment:
these are the sort of values you could create variables for - doing so would help you achieve consistency, i.e. all buttons would be the same width and height (for instance)

Reply:
We have discussed refactoring the login and register CSS files, to get rid of the repetition in both files. We will change as requested